layouts.py

```diff
@@ -43,6 +43,7 @@
     'KP_MINUS': 'NUM_MINUS',
     'KP_PLUS': 'NUM_PLUS',
     'KP_SLASH': 'NUM_SLASH',
+    'KP_PWR': 'POWER',
```

Comment:
The `KP_` keycodes are for the numeric keypad, and QMK doesn't seem to have a `KP_PWR` (i.e. what would be `KC_KP_PWR` in QMK's source; the prefix is stripped here).

Just removing this lines should be fine. I don't think `layout.py` requires any changes.

Reply:
`layouts.py` isn't actually used during build of the Configurator, it's used to generate the layouts files that are committed to the repository.

This branch is still sending the `POWER` keycode, since that's what's set in `picker.json`. You want to change that to `SYSTEM_POWER` (which `PWR` is an alias for), and remove the change in `layouts.py`.

Reply:
I've made that change though I don't have my Launch to test this right now.

Reply:
Works on my Launch